src/app/render/render.go

```diff
@@ -72,7 +72,7 @@ func Render(cCtx *cli.Context) error {
 		return fmt.Errorf("creating destination file at %q: %w", mdPath, err)
 	}
 
-	rnd := renderer.New(ch)
+	rnd := renderer.New(ch, renderer.WithDeduplicateDependencies())
```

Comment:
Since it is always enabled, should we avoid including the `renderer.WithDeduplicateDependencies()` option and make it the default behavior?